ubidots/ubidots_out.html

```diff
@@ -1,102 +1,149 @@
+       <i class="fa fa-flag"></i>
 
-    <div class="form-row">
-        <label for="node-input-name">
-            <i class="icon-tag"></i>
-            <span data-i18n="ubidots.name"></span>
-        </label>
-        <input type="text" id="node-input-name" data-i18n="[placeholder]ubidots.name" />
-    </div>
+       <span data-i18n="ubidots.select_tier"></span>
+      </label>
+       <select id="node-input-tier">
+         <option value="business">Ubidots</option>
+         <option value="educational">Ubidots for Education</option>
+     </select>
+   </div>
 
-    <div class="form-row">
-        <label for="node-input-token">
-            <i class="icon-lock"></i>
-            <span data-i18n="ubidots.token.label"></span>
-        </label>
-        <input type="text" id="node-input-token" data-i18n="[placeholder]ubidots.token.placeholder" />
-    </div>
+   <div class="form-row">
+       <label for="node-input-name">
+         <i class="fa fa-tag"></i>
+         <span data-i18n="ubidots.name"></span>
+     </label>
+       <input type="text" id="node-input-name" data-i18n="[placeholder]ubidots.name" />
+   </div>
 
-    <div class="form-row">
-        <label for="node-input-device_label">
-            <i class="icon-tag"></i>
-            <span data-i18n="ubidots.device_label.label"></span>
-        </label>
-        <input type="text" id="node-input-device_label" data-i18n="[placeholder]ubidots.device_label.placeholder" />
-    </div>
+   <div class="form-row">
+       <label for="node-input-token">
+         <i class="fa fa-lock"></i>
+         <span data-i18n="ubidots.token.label"></span>
+     </label>
+       <input type="text" id="node-input-token" data-i18n="[placeholder]ubidots.token.placeholder" />
+   </div>
+
+   <div class="form-row">
+       <label for="node-input-device_label">
+         <i class="fa fa-tag"></i>
+         <span data-i18n="ubidots.device_label.label"></span>
+     </label>
+       <input type="text" id="node-input-device_label" data-i18n="[placeholder]ubidots.device_label.placeholder" />
+   </div>
+  <div class="form-row">
+    <input type="checkbox" checked id="node-input-tls_checkbox" style="display: inline-block; width: auto; vertical-align: top;"></label>
+    <label for="node-input-tls_checkbox" style="width: calc(100% - 170px); margin-left:15px" data-i18n="ubidots.tls.label"></label>
+   </div>
+
+
+   </div>
 </script>
 
 <script type="text/x-red" data-help-name="ubidots_out">
-    <p>Receives a message and uses its values to make a request to the Ubidots API.</p>
+    <p>Receives a message and uses its values to publish a request to the Ubidots API via MQTT.</p>
 
     <p>
-    Uses the <a href="https://github.com/mqttjs/MQTT.js" target="_blank">MQTT</a> to make the connection and publish
-    the packet.
+        Uses the <a href="https://github.com/mqttjs/MQTT.js" target="_blank">MQTT</a> library to make the connection and publish the packet.
```

Comment:
Set a `rel="noopener noreferrer"` property to prevent cross-sites from using `window.opener` property and isolating the process.